README.md

```diff
@@ -4,10 +4,58 @@ A syntax tidy-upper (formatter) for PureScript.
 
 ## Install
 
+
+##### NPM
+
 ```console
 $ npm install -g purs-tidy
 ```
 
+##### [Easy PureScript Nix](https://github.com/justinwoo/easy-purescript-nix)
+
+Nix [Flakes](https://nixos.wiki/wiki/Flakes) `devShell` example (though Flakes aren’t required):
```

Comment:
I feel like this might be better-suited as just a link to `easy-purescript-nix` that states that `purs-tidy` is included there rather than the full dev shell example.

Reply:
I can understand that perspective. It could also be "hidden" behind `<summary>` / `<details>` if it's too long. 

I'm gonna go to bed, but I'll read and correct anything in the morning.